src/flisp/julia_extensions.c

```diff
@@ -171,7 +171,18 @@ static int never_id_char(uint32_t wc)
           (wc < 0xff &&
            cat >= UTF8PROC_CATEGORY_PD && cat <= UTF8PROC_CATEGORY_PO) ||
 
-          wc == '`');
+          wc == '`' ||
+
+          // mathematical brackets
+          (wc >= 0x27e6 && wc <= 0x27ef) ||
+          // angle, corner, and lenticular brackets
+          (wc >= 0x3008 && wc <= 0x3011) ||
+          // tortoise shell, square, and more lenticular brackets
+          (wc >= 0x3014 && wc <= 0x301b) ||
+          // some fullwidth punctuation
+          (wc >= 0xff08 && wc <= 0xff0f) ||
```

Comment:
Are we 100% sure that something like `＋` (0xff0b)` will never be a valid identifier like `+`?  I'd suggest leaving out the non-bracket fullwidth chars to be conservative.

Reply:
Ah, right. I forgot that we `or` `opchar?` and `never-identifier-char?` separately.